### ISSUE TITLE
Render error in curation view for cross-sentence spans with empty line

### ISSUE THREAD

**Issue by mbugert** [inception-project/inception#3175]
### Describe the bug

Switching sentences in the curation view fails with the error message `Unable to render: Position [0] is not in any row` if one goes near a cross-sentence annotation that includes an empty line.

### To Reproduce

1. create new project
2. import the attached document as UIMA CAS XMI (XML 1.0)
3. create new span annotation layer with enabled cross-sentence annotation
4. annotate a span that includes an empty line (for example `Lorem\n\nipsum`)
5. mark document as finished
6. enter curation view, set number of rows shown to 5
7. in the left sidebar, click on row 8
8. see `Unable to render: Position [0] is not in any row`

### Expected behavior

No error.

### Screenshots

![error_message](https://user-images.githubusercontent.com/23331603/177410329-fd57b64b-6329-4470-96cc-4a23d41160ec.png)

### Environment

Version and build ID: `INCEpTION -- 23.8 (2022-06-29 21:24:41, build 4b271961) `
Operating system: Linux Mint 20.3
Browser: FF 102

### Additional context
[exported-project.zip](https://github.com/inception-project/inception/files/9049498/exported-project.zip)
[document.zip](https://github.com/inception-project/inception/files/9049499/document.zip)

**Comment by reckart** [comment]
This problem is usually caused by parts of the text not being covered by sentences. Could you  have a look at the latest 24.0-SNAPSHOT and run the CAS doctor there on your file? It includes new checks which should point out the problem in the imported CAS files.

**Comment by mbugert** [comment]
I see. I ran the CAS doctor check in 23.8 and it reports `ERROR 	NoZeroSizeTokensAndSentencesCheck 	Sentence with illegal span: Sentence:33 sofa: _InitialView begin: 274 end: 274 id: <null>`. When I run the repair, both the empty lines and the reported problem are gone.

However, I would like to keep the empty lines if possible. The documents are OCR'd scans, and removing empty lines makes side-by-side comparisons with the scanned material more difficult because the document layout has changed.

I tried adding a space character to each empty line, which works (same document layout, no error in curation). This workaround is good enough for me. What remains is the philosophical question of whether empty lines should be considered illegal (I would say no, but years of webanno/INCEpTION releases disagree :wink:). Thanks!
